src/applications/financial-status-report/pages/expenses/repayments/installmentContracts.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+export const uiSchema = {
+  'ui:title': 'Your installment contracts and other debts',
+  'ui:options': {
+    classNames: 'repayments',
+  },
+  questions: {
+    hasRepayments: {
+      'ui:title':
+        'Do you make monthly payments on any installments contracts or other debts you make monthly payments on?',
+      'ui:required': () => true,
+      'ui:description': () => (
+        <>
+          Examples include:
+          <ul>
+            <li>Medical bills</li>
+            <li>Student loans</li>
+            <li>Auto loans</li>
+            <li>Home loans</li>
+            <li>Personal debts</li>
+          </ul>
```

Comment:
I noticed this has a little gap in the mockup between maybe add a `<div className="vads-u-margin-y--2">`